src/v2/providers/https.ts

```diff
@@ -41,28 +41,57 @@ export { Request, CallableRequest, FunctionsErrorCode, HttpsError };
  * Options that can be set on an individual HTTPS Cloud Function.
  */
 export interface HttpsOptions extends Omit<GlobalOptions, 'region'> {
-  /* HTTP functions can override and specify more than one regions. */
+  /** HTTP functions can override global options and can specify multiple regions to deploy to. */
   region?: SupportedRegion | string | Array<SupportedRegion | string>;
+  /** If true, allows CORS on requests to this function.
+   * If this is a String or ReqExp, allows requests from domains that match the provided string or RegExp.
+   * If this is an Array, allows requests from domains matching at least one entry of the array.
+   * Defaults to true for {@link CallableFunction}s.
```

Comment:
Is it obvious from the code that it defaults to "false" for HTTP triggers, or should we add that?